src/Editor/index.js

```diff
@@ -473,6 +479,13 @@ export class Editor extends React.Component {
         }
     }
 
+    _onKeyPress = event => {
```

Comment:
Try to be consistent with the naming. In this file they are not using a leading underscore to indicate private methods.

Reply:
Will fix!